### PR TITLE
[LayerV1] Delete LayerV1

### PR DESCRIPTION
**Pull request by kparichay**
This patch deletes LayerV1 headers and its implementations.
Some of the relevant codes from LayerV1 are moved to either LayerNode or
to LayerDevel.

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
Great! Finally, you can merge the branches!

File: nntrainer/layers/layer_devel.h [hunk 1]
```diff
@@ -50,6 +50,85 @@ enum class ExportMethods;
 class Layer {
 
 public:
+  /**
+   * @brief     Property Enumeration
+   *            0. input shape : string
+   *            1. bias zero : bool
```
Comment (lhs8928) [inline nntrainer/layers/layer_devel.h:56]: from 1 to 9 documentation doesn't match. (bias zero should be move to 9)

**Review by lhs8928 — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM. Now we upgrade Layer Implementation. We need to check once again the unit test and applications once again including accuracy, even if the unit test passed by ci but we need to check one by one for sure.

**Comment by kparichay** [comment]
> LGTM. Now we upgrade Layer Implementation. We need to check once again the unit test and applications once again including accuracy. Even if the unit test passed by ci automatically, but we need to check one by one for sure.

Agreed, I will run a manual check for all the applications, before making the layer_v2 -> main branch merge.

File: nntrainer/layers/layer_devel.h (hunk 1 above)
Comment (kparichay) [reply nntrainer/layers/layer_devel.h:56]: updated.

Comment (kparichay) [reply nntrainer/layers/layer_devel.h:56]: good catch.

**Comment by kparichay** [comment]
> LGTM. Now we upgrade Layer Implementation. We need to check once again the unit test and applications once again including accuracy. Even if the unit test passed by ci automatically, but we need to check one by one for sure.

This will be done in patch #1426.

**Pull request merged**